admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js

```diff
@@ -45,13 +45,13 @@ export default class OrderProductEdit {
 
   setupListener() {
     this.quantityInput.on('change keyup', (event) => {
-      const quantity = Number(event.target.value);
+      this.quantity = Number(event.target.value);
```

Comment:
Why changing in constant ?

Reply:
See my answer just above 😉 : https://github.com/PrestaShop/PrestaShop/pull/20186#discussion_r455610730